inspirehep/dojson/conferences/model.py

```diff
@@ -24,7 +24,7 @@
 
 from __future__ import absolute_import, division, print_function
 
-from ..schema import SchemaOverdo
+from ..model import SchemaOverdo
```

Comment:
Mmh `models` is used for SQLAlchemy.  But I guess there is no clash. In the end this is a model? Or more a mapping. Whatever name :dancer: